products/cloudiot/terraform.yaml

```diff
@@ -0,0 +1,144 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+legacy_name: "cloudiot"
+overrides: !ruby/object:Overrides::ResourceOverrides
+  DeviceRegistry: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: "google_cloudiot_registry"
+    import_format: ["{{project}}/locations/{{region}}/registries/{{name}}"]
+    id_format: "projects/{{project}}/locations/{{region}}/registries/{{name}}"
+    self_link: 'projects/{{project}}/locations/{{region}}/registries/{{name}}'
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
```

Comment:
don't you also need a pre_update here in order to set the updateMask for the custom fields?

Reply:
done